blogging/blog_guidelines.md

```diff
@@ -0,0 +1,127 @@
+
+ A user visiting a blog post on Core Zowe should be able to operate all of the software described in the article through open source license.
+
+ Posts must be vendor-neutral, although the author is allowed to describe their bio in relation to their role at a commercial company, (e.g. Janet is lead systems programmer at mainframewidgets.com), including links to social media handles.
+
+ Core Zowe articles are published on [https://medium.com/zowe](https://medium.com/zowe).
+
+ Blogs that meet the criteria for Core Zowe should have the word `[Core]` as the first word of the first sentence of the article.  This allows a reader to easily see that the content relates to core Zowe functionality.
+
+### Zowe ecosystem
+
+Zowe is a tooling platform which includes a number of frameworks allowing it to be extended by Independent Software Vendors (ISVs) or System Integrators (SIs).  Zowe extension offerings that are proven to meet the acceptance criteria for interoperating with Zowe are awarded a Zowe conformance badge, see [Zowe conformance](https://www.openmainframeproject.org/projects/zowe/conformance).
+
+Articles written about the Zowe ecosystem are welcome as they show the diversity of the platform and the capabilities a mainframe user can enjoy when using Zowe alongside one or more of its conformant products.
+
+The post may mention a vendor's name as it relates to Zowe, project deployments, adoption paths, their hosting of an in-person event or speaking at an event including virtual events, indications of meaningful participation in the community, but it shouldn't feel like an advertisement for services of your company.  
+
+Content that discusses a vendor's commercial offering should be in relation to Zowe, although background for the commercial offering may be included as long as it does not exceed 15% of the article content. Discussion of the commercial offering should be valuable to set the scene for the technical offering and its usage in relation to being exploited through use of the Zowe conformant scenarios.  Relevant descriptions of vendor technology with hyperlinks to further material hosted on other sites, including how a reader can find out more about the commercial offering, are encouraged as long as they do not become an advertisement. It is encouraged that the linked to sites hosted by companies for readers to find out more information on the vendor offering mention Zowe with correct trademark attribution.  
```

Comment:
From my experience, have any sort of % marker tends to be something that gets one into subjective arguments and ends up not being as effective of hitting your larger goal of ensuring the content isn't purely promotional.

Looking the rest of the paragraph, I think that speaks really well to the content and topic expected.